lib/processPbrMetallicRoughness.js

```diff
@@ -41,23 +41,15 @@ function processPbrMetallicRoughness(gltf, options) {
         // Pre-processing to assign skinning info and address incompatibilities
         splitIncompatibleSkins(gltf);
 
-        if (!defined(gltf.techniques)) {
-            gltf.techniques = [];
-        }
-        var materials = [];
         ForEach.material(gltf, function(material) {
             if (defined(material.pbrMetallicRoughness)) {
                 var pbrMetallicRoughness = material.pbrMetallicRoughness;
                 var technique = generateTechnique(gltf, material, options);
 
-                var newMaterial = {
-                    values : pbrMetallicRoughness,
-                    technique : technique
-                };
-                materials.push(newMaterial);
+                material.values = pbrMetallicRoughness;
+                material.technique = technique;
```

Comment:
Seems to work as advertised, but should this also be removing the `pbrMetallicRoughness` field?
Otherwise it ends up in the output despite (from what I can tell) not being needed, since the resulting gltf bundles techniques, programs, and shaders.

Is the intent to retain compatibility of the output with other engines? Or is this just for Cesium?

Reply:
Oh that's just an oversight on part, will delete `pbrMetallicRoughness`